device/CMakeLists.txt

```diff
@@ -1,20 +1,18 @@
 
 set(UMD_DEVICE_SRCS
     architecture_implementation.cpp
-    blackhole_implementation.cpp
     cpuset_lib.cpp
-    grayskull_implementation.cpp
     tlb.cpp
     tt_cluster_descriptor.cpp
     tt_device.cpp
-    tt_emulation_stub.cpp
     tt_silicon_driver.cpp
     tt_silicon_driver_common.cpp
     tt_soc_descriptor.cpp
-    tt_versim_stub.cpp
```

Comment:
Is this intentional? Along with tt_emulation_stub.cpp?

Reply:
Yes, they're not functional so will move to a deprecated folder for now.